python/trezorlib/tests/device_tests/test_msg_binance_sign_tx.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+
+from trezorlib import binance, debuglink
+from trezorlib.tools import parse_path
+
+from .common import TrezorTest
+
+
+@pytest.mark.binance
+@pytest.mark.skip_t1  # T1 support is not planned
+class TestMsgBinanceSignTx(TrezorTest):
+    def test_binance_sign_cancel_tx(self):
+        fixtures = [
```

Comment:
instead make the fixtures top-level and put the json data together with the result.
then `parametrize` the testcase.

see `unit_tests/test_cosi.py`, `test_single_ecdsa_vector` to see what I mean